srcpkgs/brother-brscan5/INSTALL.msg

```diff
@@ -0,0 +1,2 @@
+In order to access networked scanners and multifunctionals, the SANE backend
+must be configured using 'brsaneconfig5'.
```

Comment:
make this `files/README.voidlinux` and use `vdoc` to install it

Reply:
Done.

Reply:
Wasn't sure if I should do this in `post_install`, which is how many other packages do it, but those don't have a `do_install` section which this one does. I ended up putting it in `do_install` which is hopefully the right one haha. If not I can move it quickly.